source/renderer/app/utils/shelleyTrezor.ts

```diff
@@ -1,4 +1,8 @@
 import { utils } from '@cardano-foundation/ledgerjs-hw-app-cardano';
+import {
+  CardanoAddressType,
+  CardanoCVoteRegistrationFormat,
+} from '@trezor/transport/lib/types/messages';
```

Comment:
Question - could we import from the root of the library instead?
```suggestion
import { Messages } from '@trezor/transport';
```

Then we could refer to the types as follows:
```
Messages.CardanoAddressType
```
```
Messages.CardanoCVoteRegistrationFormat
```

If we import from library internals that is not considered the public API, our code can break much easier even with minor/patch updates.

Second question, `@trezor/transport` is not listed in our `package.json`. It seems like a dependency of dependency. Should we add it?

Reply:
@marcin-mazurek 
1. I totally agree with you. Done in [0d150c9](https://github.com/input-output-hk/daedalus/pull/3127/commits/0d150c9f3f96d7eda0e527a093415ff16b2b65ae)
2. I think we don't need to list it since it is `@trezor/connect` subdependency

Reply:
@DominikGuzei what do you think about point 2 ? 🙏

Reply:
My point was that dependencies of dependencies are not part of public API and could also change without notice

Reply:
but then the types will be changed in the main / parent dependency since it is used there as well, and we will see that, right? I mean, I don't want to add one more lib if that is not necessary, and if we already have access to it. Also, I agree that is safer to list it and to use it directly. Now we should see what are pros and cons since we are using only types from it...

Reply:
So I just executed `yarn add @trezor/transport@^1.1.4` on the branch, which added an appropriate entry to `package.json` but made no changes to `yarn.lock`. So I think it doesn't have any impact/side effect besides making it clear in `package.json` that it is our direct dependency. If I use `yarn add @trezor/transport@1.1.4` instead (without `^` - which is our preferred way of adding dependencies to Daedalus), it adds one minor entry to `yarn.lock` which should not affect anything either.

It's a really minor thing though and I'll let Dominik speak now.

Reply:
but yeah, let's hear from @DominikGuzei before the final decission :)

Reply:
I think if we are using it directly in our code (importing) then it's better to list it explicitly 👍

Reply:
@marcin-mazurek @DominikGuzei done in [f9fbdb9](https://github.com/input-output-hk/daedalus/pull/3127/commits/f9fbdb90cd99e40cc17fe937dbedb50205fc9924)